scripts/generator.py

```diff
@@ -2,6 +2,8 @@
 import glob
 import os
 import yaml
+import copy
```

Comment:
Did these `import` statements end up unnecessary?

Reply:
Yeah this one was unnecessary, I moved the copying from generator.py to subset_filter.py